### PR TITLE
Branded thread mailer

### PR DESCRIPTION
**Pull request by HSalmon**
[Trello card](https://trello.com/c/pENVlPL3/1496-13-branded-thread-mailer)

Previews in [this thread](https://www.loomio.org/d/81xONmLX/branded-emails-thread-mailer)

### CODE REVIEW

File: app/views/thread_mailer/_vote_buttons.html.haml [hunk 1]
```diff
@@ -1,15 +1,28 @@
...
-= link_to t(:"position_verbs.no").capitalize,
-          motion_url(motion, @utm_hash.merge(position: :no)),
-          class: 'no-link'
-= " - "
-= link_to t(:"position_verbs.block").capitalize,
-          motion_url(motion, @utm_hash.merge(position: :block)),
-          class: 'block-link'
+.thread-mailer__vote-button
+  %div
+    = image_tag("thread_mailer/yes-24.png", alt: "yes icon")
```
Comment (gdpelican) [inline app/views/thread_mailer/_vote_buttons.html.haml]: Let's translate these alt texts?

File: app/views/thread_mailer/new_discussion.html.haml [hunk 2]
```diff
@@ -1,8 +1,6 @@
...
+  %p= t :"email.create_discussion.intro", who: @discussion.author.name, which_group: @discussion.group.full_name
+  %h4= link_to @discussion.title, discussion_url(@discussion, @utm_hash)
 
-%p
-  %strong= link_to @discussion.title, discussion_url(@discussion, @utm_hash)
-
-= render_rich_text(@discussion.description, @discussion.uses_markdown)
-
-= render "footer", link: discussion_url(@discussion, @utm_hash)
+  = render_rich_text(@discussion.description, @discussion.uses_markdown)
```
Comment (gdpelican) [inline app/views/thread_mailer/new_discussion.html.haml]: Do we have a way of rendering discussion attachments in new discussion emails?

Comment (HSalmon) [inline app/views/thread_mailer/new_discussion.html.haml]: Hopefully with the last commit I made!

**Comment by robguthrie** [comment]
I'm gonna GREEEN APPLE this, maybe we can deploy it now, @HSalmon any objections, concerns?

**Pull request merged**